topics/proteomics/tutorials/DIA_lib_OSW/tutorial.md

```diff
@@ -297,7 +298,7 @@ Example: By applying an FDR of 1 % we only allow for e.g only 1 such decoy trans
 >
 > > ### {% icon solution %} Solution
 > >
-> > 1. The final spectral library contains 56.789 peptides covering 10.232 proteins.
+> > 1. The final spectral library contains over 56.000 peptides covering over 10.200 proteins.
 > > 2. Since we added equal numbers of *decoy* sequences we expect to have only half of the peptides and proteins (~28.000 and ~5.100) which are real targets.
```

Comment:
> > 2. Since we added equal numbers of *decoy* sequences we expect to have only half of the peptides and proteins (~28,000 and ~5,100) which are real targets.